### PR TITLE
Dimensional Cassandra Jolokia Metrics

### PR DESCRIPTION
**Pull request by tsheasha**
@fede1024 Please have a look and waiting for your +1

### CODE REVIEW

File: src/diamond/collectors/jolokia/cassandra_jolokia.py [hunk 1]
```diff
@@ -60,6 +63,55 @@ def update_config(self, config):
...
+            self.dimensions = {}
+            return
+
+        if key.lower() == 'count':
+            self.publish_cumulative_counter(metric_name, value)
+        else:
+            self.publish(metric_name, value)
+
+    def parse_meta(self, meta):
+        result = {}
```
Comment (fede1024) [inline src/diamond/collectors/jolokia/cassandra_jolokia.py]: What about having `dimensions` instead of `result`.

**Comment by fede1024** [comment]
+1

File: src/diamond/collectors/jolokia/cassandra_jolokia.py [hunk 2]
```diff
@@ -60,6 +63,55 @@ def update_config(self, config):
         if 'histogram_regex' in config:
             self.histogram_regex = re.compile(config['histogram_regex'])
 
+    def collect_bean(self, prefix, obj):
+        for k, v in obj.iteritems():
+            if type(v) in [int, float, long]:
```
Comment (poros) [inline src/diamond/collectors/jolokia/cassandra_jolokia.py]: Please use
```python
if isinstance(v, (int, float, long):
```
It applies also to lines below

Comment (tsheasha) [inline src/diamond/collectors/jolokia/cassandra_jolokia.py]: I agree that is a better way to write it, i took it from https://github.com/Yelp/fullerite/blob/master/src/diamond/collectors/jolokia/jolokia.py#L189-L196

I will change this though!

Comment (tsheasha) [inline src/diamond/collectors/jolokia/cassandra_jolokia.py]: fixed!

**Comment by poros** [comment]
+1

**Pull request merged**